card/src/main/java/com/adyen/checkout/card/CardAddressConfiguration.kt

```diff
@@ -7,21 +7,21 @@ import kotlinx.parcelize.Parcelize
  * Configuration class for Address Form in Card Component. This class can be used define the
  * visibility of the address form.
  */
-sealed class AddressConfiguration : Parcelable {
+sealed class CardAddressConfiguration : Parcelable {
```

Comment:
Can't we use `AddressConfiguration` as the name and it would get resolved because it's in the `card` package?

Reply:
Do you think we don't need to add prefix like "payment Method Name- Address Configuration?" . All payment method which use a address configuration can have the same name but they should be in a different package, right?